source/_includes/puppet-collections/_puppet_collections_intro.md

```diff
@@ -2,6 +2,31 @@
 > -   For Puppet 3.8 open source packages, see its [repository documentation](/puppet/3.8/puppet_repositories.html).
 > -   For Puppet Enterprise installation tarballs, see its [installation documentation](/pe/latest/install_basic.html).
 
-Puppet maintains official package repositories for several operating systems and distributions. To make the repositories more predictable, we version them as "Puppet Collections" --- each collection has all of the software you need to run a functional Puppet deployment, in versions that are known to work well with each other. Each collection is opt-in, and if you use `ensure => latest` to manage your Puppet packages, you’ll get the latest compatible versions in the collection you’re using. Whenever we make significant breaking changes that introduce incompatibilities between versions of our software, we make a new collection so that you can perform safe major upgrades at your convenience.
+Puppet maintains official package repositories for several operating systems and distributions. To make the repositories more predictable, we version them as "Puppet Collections" --- each collection has all of the software you need to run a functional Puppet deployment, in versions that are known to work well with each other. Each collection is opt-in, and you must choose a collection (and on some operating systems, install a package on Puppet-managed systems) to install software and receive updates from that collection.
 
-We maintain Puppet Collection repositories throughout the corresponding operating systems' lifespan, plus one additional month.
\ No newline at end of file
+Collection repositories are organized into two tiers that correspond to Puppet Enterprise releases, which are downstream from the collection's open-source components:
+
+-   **Numbered collections, such as Puppet Collection 1 (PC1),** are long-lived, stable repositories from which Long-Term Support (LTS) Puppet Enterprise releases are built. Numbered collections maintain the same major version of each component package during its lifetime, which delivers bug fixes and minimizes breaking changes, but also introduces fewer new features.
```

Comment:
I clicked through to the PE lifecycle link below, and we refer to it as "long term support" there, all lowercase and no hyphen. Whichever way is correct, we should be consistent between the two.